PocketKit/Sources/PocketKit/MyList/CompactMyListContainerCoordinator.swift

```diff
@@ -167,19 +171,32 @@ class CompactMyListContainerCoordinator: NSObject {
 }
 
 extension CompactMyListContainerCoordinator: UINavigationControllerDelegate {
-    func navigationController(_ navigationController: UINavigationController, didShow viewController: UIViewController, animated: Bool) {
+    func navigationController(_ navigationController: UINavigationController, willShow viewController: UIViewController, animated: Bool) {
         guard viewController == containerViewController else {
             return
         }
 
-        model.archivedItemsList.selectedReadable = nil
-        model.savedItemsList.selectedReadable = nil
+        model.archivedItemsList.selectedItem = nil
+        model.savedItemsList.selectedItem = nil
```

Comment:
didn't see any issues with setting both the readable and the web view type as nil here so simplified with just setting `selectedItem` as nil